src/channel.rs

```diff
@@ -54,6 +54,16 @@ impl<'a> Channel<'a> {
     pub fn write(&mut self, binary: bool, buf: &[u8]) -> Result<usize, RtcError> {
         Ok(self.rtc.sctp.write(self.sctp_stream_id, binary, buf)?)
     }
+
+    /// Get the amount of buffered data.
+    pub fn buffered_amount(&mut self) -> Result<usize, RtcError> {
+        Ok(self.rtc.sctp.buffered_amount(self.sctp_stream_id)?)
+    }
+
+    /// Set the threshold to emit an [`Event::ChannelBufferedAmountLow`][crate::Event::ChannelBufferedAmountLow]
+    pub fn set_buffered_amount_low_threshold(&mut self, threshold: usize) -> Result<(), RtcError> {
+        Ok(self.rtc.sctp.set_buffered_amount_low_threshold(self.sctp_stream_id, threshold)?)
+    }
```

Comment:
This shouldn't be runtime config on `Channel`, but instead options set in the `ChannelConfig` struct.